plugins/cratedb/provisioner.go

```diff
@@ -0,0 +1,44 @@
+package cratedb
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+var argsToProvision = []string{
+	"--username", fieldname.Username,
+	"--hosts", fieldname.Host,
+}
+
+type crateArgsProvisioner struct {
+}
+
+func crateProvisioner() sdk.Provisioner {
+	return crateArgsProvisioner{}
+}
+
+func (p crateArgsProvisioner) Provision(ctx context.Context, in sdk.ProvisionInput, out *sdk.ProvisionOutput) {
+	if value, ok := in.ItemFields[fieldname.Password]; ok {
+		out.AddEnvVar("CRATEPW", value)
+	}
+	
+	for i, arg := range argsToProvision {
```

Comment:
Please replace this with the latest comment: https://github.com/1Password/shell-plugins/pull/276#discussion_r1244985107

Reply:
```sql
var user, host string
	if fieldValue, ok := in.ItemFields[fieldname.Username]; ok {
		user=fieldValue
	}
	if fieldValue, ok := in.ItemFields[fieldname.Host]; ok {
		host=fieldValue
	}
		commandLine := []string{out.CommandLine[0], "--username", user, "--hosts", host, }
		commandLine = append(commandLine, out.CommandLine[1:]...)
		out.CommandLine = commandLine
	```

Like this